modules/ppcp-axo/resources/js/AxoManager.js

```diff
@@ -584,6 +586,7 @@ class AxoManager {
             const authResponse = await this.fastlane.identity.triggerAuthenticationFlow(lookupResponse.customerContextId);
 
             log('AuthResponse', authResponse);
+            await this.log(`triggerAuthenticationFlow: ${JSON.stringify(authResponse)}`);
```

Comment:
I don't like the fact that now we have both `log()` and `this.log` in the code - it's confusing.

Can we perhaps combine these, to keep the code cleaner, but still keep the `console.log` functionality and also add the WooCommerce logger.

Reply:
Yeah, I don't like it either... as mentioned in a previous comment I don't know exactly which information should we log so for now I just ignored the existing console log to avoid noise if the WC logs, but yes the idea is to consolidate both in a single one.